spec/bigcommerce/lightstep/interceptors/env_spec.rb

```diff
@@ -44,6 +44,14 @@
           expect(span).to receive(:set_tag).with('provider.datacenter', 'us-central1-a').once.ordered
           subject
         end
+
+        it 'should only collect the tags once even with multiple calls' do
+          expect(env).to receive(:fetch).with('GIT_SHA', nil).once.ordered
+          expect(env).to receive(:fetch).with('DATACENTER', nil).once.ordered
+          subject
+          subject
```

Comment:
Isnt subject memoized within a test?

Reply:
🤔 I didn't think so, but worth adjusting to be entirely sure.

Reply:
https://relishapp.com/rspec/rspec-core/docs/subject/explicit-subject#the-%60subject%60-is-memoized-within-an-example-but-not-across-examples

Reply:
TIL! Thanks!